Kernel/CMakeLists.txt

```diff
@@ -342,17 +335,25 @@ set(KERNEL_SOURCES
     TTY/SlavePTY.cpp
     TTY/TTY.cpp
     TTY/VirtualConsole.cpp
+    Tasks/Coredump.cpp
+    Tasks/CrashHandler.cpp
     Tasks/FinalizerTask.cpp
+    Tasks/FutexQueue.cpp
+    Tasks/PerformanceEventBuffer.cpp
+    Tasks/Process.cpp
+    Tasks/ProcessExposed.cpp
+    Tasks/ProcessGroup.cpp
+    Tasks/ProcessProcFSTraits.cpp
+    Tasks/ProcessSpecificExposed.cpp
+    Tasks/Scheduler.cpp
     Tasks/SyncTask.cpp
-    Thread.cpp
-    ThreadBlockers.cpp
-    ThreadTracer.cpp
+    Tasks/Thread.cpp
+    Tasks/ThreadBlockers.cpp
+    Tasks/ThreadTracer.cpp
```

Comment:
this might belong more to `Debug` despite the name (or rather: due to name?)…

Reply:
I don't want to create a new subdirectory called `Debug` as this name feels awkward to me, and it belongs to to the task-managing code...